test/langtools/jdk/javadoc/doclet/testMethodCommentAlgorithm/TestMethodCommentsAlgorithm.java

```diff
@@ -418,41 +387,67 @@ public interface T5 {
     }
 
     /*
-     * Takes a path to the java.base module, finds the Object.java file in
-     * there, creates a copy of that file _with the modified doc comment_
-     * for Object.equals in the provided destination directory and returns
-     * the path to that created copy.
+     * Locates source of the java.base module.
      */
-    private Path createPatchedJavaLangObject(Path src, Path dst, String newComment)
-            throws IOException {
-        if (!Files.isDirectory(src) || !Files.isDirectory(dst)) {
-            throw new IllegalArgumentException();
+    private Path findJavaBase() {
+        String testSrc = System.getProperty("test.src");
+        if (testSrc == null) {
+            // shouldn't happen
```

Comment:
Purely FYI, it could happen if the class is run directly, without `jtreg`.
That being said, very few people do that these days.
No change necessary,